articles/quickstarts/index.md

```diff
@@ -19,17 +19,144 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
 - APIs for Python and .NET languages (C#, F#, and VB.NET) for running quantum programs written in Q#
 - Tools to facilitate your development
 
-Q# programs can run as standalone applications using Visual Studio Code or Visual Studio, or through Jupyter Notebooks with the IQ# Jupyter kernel.
-They can also be paired with a host program written in a .NET language (typically C#) or Python, enabling you to call quantum operations from inside a classical program.
+Q# programs can run as standalone applications using Visual Studio Code or Visual Studio, through Jupyter Notebooks with the IQ# Jupyter kernel, or paired with a host program written in a .NET language (typically C#) or Python. You can also run Q# programs online using [Codespaces](https://online.visualstudio.com/) or [MyBinder.org](https://mybinder.org/). 
 
-The workflows for each of these setups are described and compared at [Ways to run a Q# program](xref:microsoft.quantum.guide.host-programs).
+## Installation options
+
+You can use the QDK three ways:
+
+- [Install the QDK locally](#install-the-qdk-locally)
+- [Use the QDK online](#use-the-qdk-online)
+- [Use a QDK Docker image](#use-the-qdk-with-docker)
+
+## Install the QDK locally
```

Comment:
For each one of the options under here, please add an additional step linking to the getting started example relevant for this environment. 
Same for the online options

Reply:
Fixed